### ISSUE TITLE
Softmax layer  (page 437)

### ISSUE THREAD

**Issue by ashwinshetgaonkar** [rasbt/machine-learning-book#28]
Here you have not defined the softmax layer while building the Sequential model.

Is is taken by default when we choose loss to be:nn.CrossEntropyLoss()?
